magento/magento2
# MAGETWO-5015: Report error csv doesn't work when trying to import a csv file with semicolon delimiter.

### Description
Product import error report taking into consideration value delimiter.

### Fixed Issues (if relevant)
1. magento/magento2#5015: Report error csv doesn't work when trying to import a csv file with semicolon delimiter

### Manual testing scenarios
1. Try to import a catalog_product.csv file in admin interface which contains an error (try to insert incorrect visibility field value, for example 'cathaloig').
2. Configure the semicolon (';') for row delimiter.
3. Click Check data button, the file is valid for import.
4. You should be able to download the report error csv file.

### Contribution checklist
